packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/index.js

```diff
@@ -29,9 +30,13 @@ const CellContent = ({ content, fieldSchema, metadatas, name, queryInfos, rowId
 
       return <MultipleMedias value={content} />;
 
-    case 'relation':
+    case 'relation': {
+      if (['oneToOne', 'manyToOne'].includes(fieldSchema.relation)) {
```

Comment:
@soupette Looking at the codebase I see two different definitions of this pattern:

- ['oneWay', 'oneToOne', 'manyToOne', 'oneToManyMorph', 'oneToOneMorph']
- ['oneWay', 'oneToOne', 'manyToOne']

Are we missing something here? What do you think about creating a new utility ex: `isSingleRelation(type)` and to re-use this (in a different PR) were possible? I can see at least 5 - 10 different locations with different patterns...

Reply:
You are right !

Currently we need the `<**>Morph` in order to generate the yup schema, otherwise these relations are not displayable in the CM edit view. 
However in the CM list view we can display `oneToOneMorph` only.

Reply:
Nice, thanks. I've added `oneToOneMorph` in 949d3f68ebc2d2ba43f25a09e916c5a9bfaed310 and extracted the array into a local utils function for now.